.github/workflows/ci.yml

```diff
@@ -116,6 +116,40 @@ jobs:
           PROFILES: ${{ matrix.PROFILES }}
           SHELL: "/bin/bash"
 
+  test_runtime_benchmarks:
+    name: Test runtime benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the source code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Install latest nightly
```

Comment:
Why install nightly? For the compile-time benchmarks we only install beta.

Reply:
The function that we use to gather a toolchain for runtime benchmarks tries to find a nightly toolchain using rustup, if it's not available, it fails.. I'm not sure if it's required, but I didn't want to change existing code. But it's not needed for check, so I'll remove it for now.